### DISCUSSION TITLE
MacOS Zoom Accessibility Support

### DISCUSSION THREAD

**Discussion by Acrylic125** [ghostty-org/ghostty#3852] [Q&A]
I rely on magnifying tools to do my work due to me being visually impaired. I primarily work on MacOS so I use the inbuilt Zoom Accessibility feature. 
![image](https://github.com/user-attachments/assets/bc6cce81-b468-4d68-9fd9-8f964347def1)

One of the key features of this tool is to follow my text cursor. This is incredibly useful for my terminal based workflows (including using neovim) since I only need to type on my keyboard without having to move my mouse to focus in on the parts I am typing.
![image](https://github.com/user-attachments/assets/2b246d46-1bf3-4568-894f-b349e167d90a)

Unfortunately, unlike other terminals like Mac's inbuilt terminal and iTerm, the default Ghostty setup does not seem to provide such support. 

Any workarounds to support this use case?

**Comment by jsjoeio** [comment]
Is this the same as relying on `cmd` + `+` to zoom in? that's what I rely on to change the size of the font in the terminal and noticed it doesn't work in Ghostty

**Reply by Acrylic125** [reply]
Nope, it is different!

if enabled, the default hotkey is `CMD` + `Opt` + `+`/`-`. It zooms in your entire screen without affecting the zoom state of the app. https://support.apple.com/en-sg/guide/mac-help/mchl779716b8/mac 

Anyways, `CMD` + `+/-` works for me tho on a clean install of Ghostty.

**Reply by jsjoeio** [reply]
ahh, i see the difference now. thanks!

nice! maybe i need a fresh install then
